services/ux-backend/main.go

```diff
@@ -94,3 +104,26 @@ func main() {
 	log.Fatal(err)
 
 }
+
+func newClient() (client.Client, error) {
+	klog.Info("Setting up k8s client")
+	scheme := runtime.NewScheme()
+	if err := corev1.AddToScheme(scheme); err != nil {
+		return nil, err
+	}
+	if err := ocsv1.AddToScheme(scheme); err != nil {
+		return nil, err
+	}
+
+	config, err := config.GetConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	client, err := client.New(config, client.Options{Scheme: scheme})
+	if err != nil {
+		return nil, err
+	}
+
+	return client, nil
+}
```

Comment:
1. I believe we already have this code in the code base somewhere else. 
I would prefer to have a shared code invoked from all locations. 

2. In this file all private function resides above the main function, can we please be consistency